hircluster.h

```diff
@@ -142,6 +141,8 @@ typedef struct redisClusterAsyncContext {
     int err;          /* Error flags, 0 when there is no error */
     char errstr[128]; /* String representation of error when applicable */
 
+    int64_t update_route_time; /* Timestamp for next allowed slotmap update */
```

Comment:
The name of this variable could indicate if this is "next allowed" or "last successful" or "last attempt". It would be useful to see this from the variable name.

Btw, I suggested in another comment to store the last attempt timestamp instead. It seems more in line which last connect attempt for nodes.

Reply:
Updated to lastSlotmapUpdateAttempt